### PR TITLE
terraform-graph-beautifier: Update to version 0.3.4 and add support for arm64

### LINKED ISSUE #128: terraform-graph-beautifier: doesn't update correctly
Error from the excavator
```text
terraform-graph-beautifier: 0.3.4 (scoop version is 0.2.0) autoupdate available
Autoupdating terraform-graph-beautifier
Searching hash for terraform-graph-beautifier_0.3.4_Windows_x86_64.tar.gz in https://github.com/pcasteran/terraform-graph-beautifier/releases/download/v0.3.4/checksums.txt
Could not find hash in https://github.com/pcasteran/terraform-graph-beautifier/releases/download/v0.3.4/checksums.txt
Downloading terraform-graph-beautifier_0.3.4_Windows_x86_64.tar.gz to compute hashes!
VERBOSE: Requested HTTP/1.1 GET with 0-byte payload
VERBOSE: Received HTTP/1.1 response of content type application/json of unknown size
VERBOSE: Content encoding: utf-8
The remote server returned an error: (404) Not Found.
URL https://github.com/pcasteran/terraform-graph-beautifier/releases/download/v0.3.4/terraform-graph-beautifier_0.3.4_Windows_x86_64.tar.gz is not valid
ERROR Could not update terraform-graph-beautifier, hash for terraform-graph-beautifier_0.3.4_Windows_x86_64.tar.gz failed!
```

Investigation is required

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [x] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other
## Description
- Update the download url for 64bit which changed over the year
- Add arm64 support
## Related Tickets & Documents
Fixes #128
## [optional] What gif/emoji best describes this PR or how it makes you feel?
:alien::bug::lipstick:
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Notes in the manifest?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Dependencies?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Suggestions?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### File persistence to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### terraform-graph-beautifier

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
